src/marten/cli/manage/command/seed.cr

```diff
@@ -0,0 +1,54 @@
+module Marten
+  module CLI
+    class Manage
+      module Command
+        class Seed < Base
+          @seed_path : String = "./seed.cr"
+
+          help "Populate the database by running the seed file."
+
+          def setup
+            on_option_with_arg(
+              :f,
+              :file,
+              arg: "path",
+              description: "Specify a custom path to the seed file"
+            ) do |v|
+              @seed_path = v
+            end
+          end
+
+          def run
+            run_seed_file
+          end
+
+          private SEED_SOURCE_CONTENT = ECR.render("#{__DIR__}/new/templates/app/src/app/seed.cr.ecr")
+
+          private getter seed_path
+
+          private def run_seed_file
+            if File.exists?(seed_path)
+              print "Running seed file at #{seed_path}"
+              begin
+                Process.run("crystal #{seed_path}", shell: true, output: stdout, error: stderr)
+              rescue ex : Exception
+                print "Error running seed file: #{ex.message}"
+              end
+            else
+              print "Seed file not found at #{seed_path}. Generating default seed.cr..."
+              write_seed_file
```

Comment:
I am not sure that the `seed` command should generate anything. Ideally, I think that this command should focus on executing the specified seed file (or output an error if it does not exist) while generating seeds should be done by a dedicated generator. WDYT?

Reply:
My intention was that the user doesn't have to think about how to (re-)create a seed file if none is found so he doesn't have to look up what file content is required for seeding.

But I'm open to remove this functionality. Alternatively a dedicated flag could create it? What do you think?

Reply:
The thing is that the user might have made a typo while typing the `seed` command or incorrectly specified it. We can't really know for sure and we certainly don't want to make these kinds of assumptions if they lead to us creating files that pollute their repository or their file system (eg. if they made a type in the seed name, should we really create a corresponding file automatically?). When the specified file does not exist, I think that the best course of action is really to return a clear error that eventually mentions how to generate the seed file (through the use of a dedicated generator).

Additionally, I really want to enforce the use of generators for all our generation-related cases. So it makes sense to redirect users to this command should they need to generate seeds. For this reason, I don't think that the command should accept a generation-specific flag either: on the one hand we have a `seed` command used for running seeds, and on the other we allow generating seeds with generators.